benches/throughput.rs

```diff
@@ -79,10 +80,15 @@ fn initialize_environment() -> (Arc<Mutex<TestEnv>>, Runtime) {
         direct.set_nodelay(true).unwrap();
         info!("setup complete");
 
+        let client_mode = match mode {
```

Comment:
Shouldn't this be just `mode`, possibly with a clone?

Reply:
Its the inverse; if server is a reader then client must be a writer